bot/minibot.py

```diff
@@ -1,162 +0,0 @@
-#!/usr/bin/env python3
-# This bot was made by tkdrg.
-# Ask #coderbus@irc.rizon.net if this breaks.
-# See LICENSE-bot_folder.txt for the license of the files in this folder.
```

Comment:
Deleting minibot is premature.

if your system becomes the primary system sure, we can discuss it then it (would still be likely rejected by the maintainers since minibot is more portable then your system is), but until then it might be best to keep this around and keep the game compatible with both